pkg/ovsdb/transact.go

```diff
@@ -565,6 +582,8 @@ func (c *Condition) Compare(row *map[string]interface{}) (bool, error) {
 		return c.CompareString(row)
 	case libovsdb.TypeUUID:
 		return c.CompareUUID(row)
+	case libovsdb.TypeEnum:
```

Comment:
Why do we need to compare Enum with Condition? I think Enum is relevant as a restriction to new values only, isn't it ?

Reply:
if enum then you need to check what type it is an compare accordingly (we support only enum strings for now)